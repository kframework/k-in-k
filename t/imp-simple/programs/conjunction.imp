<k> True && False ~> .K </k>
