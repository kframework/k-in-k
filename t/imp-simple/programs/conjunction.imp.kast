kCell {  } ( kseq {  } ( inj { BExp {  } , KItem {  } } ( And {  } ( inj { BExpResult {  } , BExp {  } } ( True {  } (  ) ) , inj { BExpResult {  } , BExp {  } } ( False {  } (  ) ) ) ) , dotk {  } (  ) ) )
