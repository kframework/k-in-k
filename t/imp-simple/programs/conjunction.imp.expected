kCell { } ( kseq { }(inj{BExpResult{}, KItem{}}(False{}()), dotk{}()))
